ibis/backends/dask/execution/window.py

```diff
@@ -49,11 +184,43 @@ def _post_process_empty(
         return series[0]
     else:
         # Project any non delayed object to the shape of "parent"
-        return parent.apply(lambda row: result, meta=(None, 'object'))
+        if isinstance(parent, dd.DataFrame):
```

Comment:
In what case do we need to select the first column out first?

Reply:
Yeah, this can be simply implemented with just passing `axis=1` to the apply.
We just want to broadcast the result value across the whole parent index.